Fitness/Controllers/HabitTrackingController+Extensions.swift

```diff
@@ -3,7 +3,7 @@
 //  Fitness
 //
 //  Created by Kevin Chan on 5/14/19.
-//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
```

Comment:
lmao y was this made under keivan's name anyway

Reply:
i have no clue